src/hugo/content/en-us/patterns/MVC/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Model-View-Controller Device Software Design"
+weight: 10
```

Comment:
Can you reset the weights so that this renders alphabetically? You can set the weight between _Gateway_ and _Software Update_ values.